src/components/Home.vue

```diff
@@ -2,8 +2,275 @@
 </script>
 
 <template>
-	<h1>Home</h1>
+
+	<div id="container-and-footer">
+		<div id="container">
+			<nav>
+				<div id="links">
+					<a href="#">features</a>
+					<a href="#">pricing</a>
+					<a href="#">about</a>
```

Comment:
These are intentionally not capitalized right?

Reply:
upon looking over the rest I this does appear to be the case

Reply:
yes